src/main/java/wooteco/subway/domain/member/Favorite.java

```diff
@@ -0,0 +1,38 @@
+package wooteco.subway.domain.member;
+
+import java.util.Objects;
+
+public class Favorite {
+    private Long sourceId;
+    private Long destinationId;
+
+    public Favorite() {
```

Comment:
```suggestion
    private Favorite() {
```
기본생성자를 사용하지 않으면 private으로 막아주세요
다른 도메인 클래스도 마찬가지로 적용해주세요